### PR TITLE
[DOCS] [8.1] [BUG] Timeline schema topic - update UI and add dataViewId field

### LINKED ISSUE #1709: [BUG] Timeline Schema of Documentation requires update
## Description 

Timeline Schema of Documentation require updatation

- Old Data Source terminology expected Data View should be there
- Positioning of Timeline elements need to be updated 

## URL

https://www.elastic.co/guide/en/security/8.1/timeline-object-schema.html

## Screen-Shoot

- Actual Result

image

- Expected Result

image

### PR DESCRIPTION
**Pull request by benironside**
Fixes #1709 by adding the dataViewId field to the Timeline schema, and updating the screenshot with labeled Timeline UI elements.

The "eventType" field still exists (I exported a timeline to check), but one thing I don't understand is that - whether or not I checked "alerts only" - the value of "eventType" was "all".

Preview: [Timeline schema](https://security-docs_1727.docs-preview.app.elstc.co/guide/en/security/master/timeline-object-schema.html)

### CODE REVIEW

File: docs/reference/timeline-schema.asciidoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ timestamp.
 |[[timeline-object-desc]]`description` |String |The timeline's description.
 |[[timeline-object-event-notes]]`eventNotes` |<<eventNotes-obj, eventNotes[]>>
 |Notes added to specific events in the timeline.
-|[[timeline-object-eventtype]]`eventType` |String a|Event types displayed in
+|`eventType` |String a|Event types displayed in
 the timeline, which can be:
```
Comment (nastasha-solomon) [inline docs/reference/timeline-schema.asciidoc:54]:
```suggestion
the Timeline, which can be:
```

**Review by joepeeples — APPROVED** [review]
Looks good! A few of the items in the screenshot are a little hard to distinguish because they're so close together in the UI (1, 2, 3 and 7, 9). Maybe the numbers could be spaced out a little more, but I wouldn't lose a lot of time on it -- it's already pretty clear, especially when you compare the image with the numbered list below.

**Review by karanbirsingh-qasource — APPROVED** [review]
Approved !!

**Review by jmikell821 — APPROVED** [review]
LGTM!

**Pull request merged**